sdk/storage/azure-storage-file-share/README.md

```diff
@@ -179,9 +179,8 @@ ShareClient shareClient = new ShareClientBuilder().endpoint(shareURL)
  Once you have the SASToken, you can construct the file service client with `${accountName}`, `${shareName}`, `${directoryPath}`, `${sasToken}`
 
 ```java
```

Comment:
@gapra-msft @rickle-msft we should update these to use embedme.

Reply:
There is no `ReademeSample.java` and no `embedme` comments

Reply:
@alzimmermsft - Hi, I have add `ReadmeSample.java` and update these to use `embedme`, please review.